spec/factories/models/organisations.rb

```diff
@@ -8,5 +8,11 @@
     initialize_with do
       Organisation.create_with(govuk_content_id:, name:).find_or_initialize_by(slug:)
     end
+
+    trait :with_signed_mou do
+      after(:build) do |organisation|
+        organisation.mou_signatures << (create :mou_signature_for_organisation, organisation:)
+      end
+    end
```

Comment:
I don't see any major issues with this. But if there's likely to only be a few specs that depend on an MOU existing for the user/organisation, is it clearer to just build the `mou_signature` in the spec rather than having it as a trait?

Reply:
It's mainly to clean up tests so that they're more readable. In this case, we added a trait to the user factory which is bundled along with creating an `organisation_admin` user, and that triggers a sequence of factory traits to make sure that user is in a valid state. Doing all of that piece by piece for tests that involve an organisation admin would just get repetitive, and wouldn't result in additional coverage.